src/tracing/errors.ts

```diff
@@ -46,6 +46,18 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
      * Therefore, this can be ignored.
      */
     if (error.message.match(/Unexpected token '<'/)) return null
+
+    /*
+     * Content security policy errors can be filtered out because there are expected failures.
```

Comment:
This will not filter out all CSP errors, only `unsafe-eval`.

We _should_ be reporting these violations somewhere, but it's not yet configured. You should ping @cmcewen to see if that's planned. Once that's done, you can expand the regex to match all reported errors. Could you ticket that and link it in a TODO(INFRA-123)-style comment?